ui/helpers/httpHelper.js

```diff
@@ -0,0 +1,32 @@
+import Cookies from 'js-cookie';
+
+const getHeaders = function getHeaders() {
+  return new Headers({
+    'X-CSRFToken': Cookies.get('csrftoken'),
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+  });
+};
+
+export const postJson = function postJson(uri, data) {
```

Comment:
I considered alternate names for these functions.  Curious about your opinion:

``postJson`` === ``create``
``putJson`` === ``update``
``deleteRecord`` === ``destroy``

These mostly match the terminology for DRF...

Reply:
Sounds good to me :-)